micro-ui/web/micro-ui-internals/packages/modules/workbench/src/pages/employee/UploadBoundary.js

```
@@ -178,14 +179,14 @@ const UploadBoundary = () => {
                 label = label + t(Digit.Utils.locale.getTransformedLocale(err?.code)) + ", ";
               }
             });
-            setShowToast({ label, isError: true });
+            setShowToast({ label, type: "error" });
             closeToast();
           },
         }
       );
     } catch (error) {
       let label = `${t("WBH_BOUNDARY_UPSERT_FAIL")}: `;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'label' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)